### PR TITLE
Changed User.php in Entity directory.

### PR DESCRIPTION
**Pull request by SopelPL**

### CODE REVIEW

File: src/Entity/User.php [hunk 1]
```diff
@@ -1,108 +1,198 @@
...
+
+    /**
+     * @ORM\Column(type="boolean")
+     */
+    private $confirmed = false;
+
+    /**
+     * @ORM\Column(type="string", length=25)
+     */
+    private $discord;
```
Comment (error56) [inline src/Entity/User.php]: What?

File: src/Entity/User.php [hunk 2]
```diff
@@ -1,108 +1,198 @@
...
+
+    public function getId(): ?int
+    {
+        return $this->id;
+    }
+
+    /**
+     * @ORM\Column(type="datetime")
+     */
+    private $created_at;
```
Comment (error56) [inline src/Entity/User.php]: PSR-2

File: src/Entity/User.php [hunk 3]
```diff
@@ -1,108 +1,198 @@
...
+     * @ORM\Column(type="string", length=25)
+     */
+    private $discord;
+
+    /**
+     * @ORM\Column(type="string")
+     */
+    private $roles = [];
+
+    public function getId(): ?int
```
Comment (error56) [inline src/Entity/User.php]: This must be under `created_at` field

File: src/Entity/User.php [hunk 4]
```diff
@@ -1,108 +1,198 @@
...
+
+        return $this;
+    }
+
+    /**
+     * @see UserInterface
+     */
+    public function getEmail(): string
+    {
+        return (string) $this->email;
```
Comment (error56) [inline src/Entity/User.php]: Why are you cast this var to string?

File: src/Entity/User.php [hunk 5]
```diff
@@ -1,108 +1,198 @@
...
+
+        return $this;
+    }
+
+    /**
+     * @see UserInterface
+     */
+    public function getAvatar(): string
+    {
+        return (string) $this->avatar;
```
Comment (error56) [inline src/Entity/User.php]: cast

File: src/Entity/User.php [hunk 6]
```diff
@@ -1,108 +1,198 @@
...
+
+        return $this;
+    }
+
+    /**
+     * @see UserInterface
+     */
+    public function getConfirmed(): string
+    {
+        return (string) $this->confirmed;
```
Comment (error56) [inline src/Entity/User.php]: This isn't string, the name of function would be `isConfirmed`

File: src/Entity/User.php [hunk 7]
```diff
@@ -1,108 +1,198 @@
...
+
+    public function setAvatar(string $avatar): self
+    {
+        $this->avatar = $avatar;
+
+        return $this;
+    }
+
+    /**
+     * @see UserInterface
```
Comment (error56) [inline src/Entity/User.php]: Why?

File: src/Entity/User.php [hunk 8]
```diff
@@ -1,108 +1,198 @@
...
+
+    public function setEmail(string $email): self
+    {
+        $this->email = $email;
+
+        return $this;
+    }
+
+    /**
+     * @see UserInterface
```
Comment (error56) [inline src/Entity/User.php]: Why?

File: src/Entity/User.php [hunk 9]
```diff
@@ -1,108 +1,198 @@
...
+
+    public function setConfirmed(string $confirmed): self
+    {
+        $this->confirmed = $confirmed;
+
+        return $this;
+    }
+
+    /**
+     * @see UserInterface
```
Comment (error56) [inline src/Entity/User.php]: Why?

File: src/Entity/User.php [hunk 10]
```diff
@@ -1,108 +1,198 @@
...
+    }
+
+    public function setRoles(array $roles): self
+    {
+        $this->roles = $roles;
+
+        return $this;
+    }
+
+    public function getCreatedAt() : datetime
```
Comment (error56) [inline src/Entity/User.php]: `(): \DateTime`

File: src/Entity/User.php [hunk 11]
```diff
@@ -1,108 +1,198 @@
...
+    {
+        return $this->created_at;
+    }
+
```
Comment (error56) [inline src/Entity/User.php]: Where is created at setter?

File: src/Entity/User.php (hunk 1 above)
Comment (SopelPL) [reply src/Entity/User.php]: It's a discord tag

Comment (error56) [reply src/Entity/User.php]: Describe it. discordTag.

File: src/Entity/User.php [hunk 12]
```diff
@@ -1,108 +1,194 @@
...
+    public function setAvatar(string $avatar): self
+    {
+        $this->avatar = $avatar;
+
+        return $this;
+    }
+
+    public function isConfirmed(): boolean
```
Comment (error56) [inline src/Entity/User.php]: bool*

File: src/Entity/User.php [hunk 13]
```diff
@@ -1,108 +1,196 @@
...
+
+        return $this;
+    }
+
+    public function getCreatedAt(): \DateTime
+    {
+        return $this->createdAt;
+    }
+
+    public function setCreatedAt($createdAt): self
```
Comment (error56) [inline src/Entity/User.php]: \DateTime $createdAt

**Pull request merged**